.ruby-version

```diff
@@ -1 +1 @@
-2.3.7
+2.4.5
```

Comment:
If there's no need for newest additions in latest Ruby versions we should try to keep the Ruby version shipped with latest version of macOS (don't even know which it is and if 2.3.7 was already that) so that people not wanting to use rvm/rbenv won't have to worry. Except if you saw features in more recent Ruby versions that we needed?

Reply:
I actually tried that in this PR (and the other PR in SwiftGen), but it starts complaining about an unknown version of ruby, so 🤷‍♂️